### PR TITLE
!gamble

### PR DESCRIPTION
**Pull request by VuongRon**
# Changes

- Linted some unlinted files
- Added [sinon](https://sinonjs.org/) and [sinon-chai](https://github.com/domenic/sinon-chai) to enable local stubs/spies/watchers in tests
- added `getRandomArrayIndex` method to RNG service
- Added `!gamble` command which accepts integer inputs or 'all' as first arg
- Renamed test files to more conventional `x.test.js` for IDE recognition
- Added test coverage for `!gamble`

### CODE REVIEW

File: commands/gamble.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          `${outcomeMessage} **You lost.**`,
+          `You lost **${value}** Tsushobucks.`,
+          `Your new balance is **${user.balance}**.`,
+        ].join("\n\n");
+      }
+      await user.save().catch((err) => {
+        console.error(err);
+        return;
+      });
+      console.log("user saved");
```
Comment (dcepil) [inline commands/gamble.js]: hmm...

File: test/gamble.test.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        balance: 10,
+        save: sandbox.stub().resolves(true),
+      };
+      this.stubEmbedMessage = sandbox.stub(embedService, "embedMessage").returns(true);
+      this.stubFindOrCreateByDiscordId = sandbox.stub(userModel, "findOrCreateByDiscordId").returns(this.mockUser);
+      this.stubGetRandomInt = sandbox.stub(rngService, "getRandomInt").returns(0);
+      this.stubGetRandomArrayIndex = sandbox.stub(rngService, "getRandomArrayIndex").returns("Outcome.");
+    });
+
+    const badArgs = ["abc", "$1.99", "o10"];
```
Comment (DarkStoorM) [inline test/gamble.test.js]: Should also test against empty arg
